nextjs-task-system/context/TaskContext.tsx

```
@@ -0,0 +1,91 @@
+      user: null,
+      group: null,
+      dueDate: new Date(),
+      priority: { id: 0, name: "" },
+      status: { id: 0, name: "" },
+      creationDate: new Date(),
+      comments: [],
+    },
+  });
+  {
+    /*Values for the delete modal */
+  }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)